app/views/organisations/edit.html.erb

```diff
@@ -0,0 +1,19 @@
+<%= render "layouts/form_errors", resource: @organisation %>
+  <%= link_to "Back to settings", organisation_path, class: "govuk-back-link" %>
+  <h1 class="govuk-heading-l">Edit Organisation Settings</h1>
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <div class="govuk-form-group govuk-body-m" >
+
+        <%= form_for @organisation do |form| %>
+          <%= form.label :name, "Organisation name", class: "govuk-label" %>
+          <%= form.text_field :name, class: "govuk-input" %><br />
+
+          <%= form.label :service_email, "Service email", class: "govuk-label govuk-!-padding-top-4" %>
```

Comment:
There's a bunch of classes to organise forms that work better than these `govuk-!` classes on this form, as this form doesn't do anything particularly special.

You can see them use in the examples in the design system - like https://design-system.service.gov.uk/patterns/addresses - or elsewhere in the app. 

Wrapping label/fields in `div`s with the `govuk-form-group`, is roughly what you want.